internal/engine/experiment/stunreachability/fake_test.go

```diff
@@ -6,6 +6,9 @@ import (
 	"time"
 )
 
+// TODO(bassosimone): we should use internal/netxlite/mocks rather
+// than rolling out a custom type private to this package.
+
```

Comment:
I could have done it as part of this diff, but no. The diff is already too large.